shared/constants/fs.js

```diff
@@ -651,7 +659,7 @@ export const syntheticEventToTargetRect = (evt?: SyntheticEvent<>): ?ClientRect
 // shouldUseOldMimeType determines if mimeType from newItem should reuse
 // what's in oldItem.
 export const shouldUseOldMimeType = (oldItem: Types.FilePathItem, newItem: Types.FilePathItem): boolean => {
-  if (oldItem.mimeType === '' || newItem.mimeType !== '') {
+  if (oldItem.mimeType === null || newItem.mimeType !== null) {
```

Comment:
Slight preference for `if (!oldItem.mimeType || !new.mimeType)`

Same for other places (e.g. in `shared/fs/filepreview/view-container.js`) if you agree.

Reply:
Doesn't "!foo" mean "!(foo == true)" with all the conversions, perhaps keep '!==' there?

Reply:
Ok, I'll do it that way.